config/samples/oadp_v1alpha1_dataprotectionapplication.yaml

```diff
@@ -10,6 +10,7 @@ spec:
       - openshift
       - aws
       - kubevirt
+      - hypershift
```

Comment:
I do not know if we want this change

this is basically a suggestion we give users of an example DPA

when users nedd to use `hypershift` plugin?

Reply:
same comment applies to kubevirt tho.

Reply:
Hmm. Yeah. This is just a sample for format purposes. I think it's fine to include both here, as long as we're documenting elsewhere which plugins to use for which users. i.e. this sample shows where to enable it if you want it, somewhere else we document who wants it.

Reply:
ok, I will keep this change then. Let me know if you change your mind to remove it.